src/components/menu/menu.tsx

```diff
@@ -279,7 +279,7 @@ class MenuItem extends Component<any, any> {
         ref={element => {
           this.props.addAccessibleChild(element);
           if (props.isSelected(props.data)) {
-            props.setDefaultFocusedElement(element);
+            setTimeout(() => props.setDefaultFocusedElement(element))
```

Comment:
why does this fix the problem ?

Reply:
it not a matter of time it is a matter of the cycle of the eventl lopp, once you pass it the the setTimeout it is automatically delayed to the next loop in the event loop